### ISSUE TITLE
Inspection for isset($_SERVER[...])

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#418]
I had big problem on my code because I not checked if `$_SERVER['HTTP_ACCEPT_LANGUAGE']` does exists before use that. It because on a browser it always exists, but from a bot or an API access it is optional. I think that we should create an additional inspection to check if it is tested before be used.

```php
// Warning: HTTP_ACCEPT_LANGUAGE could not exists on $_SERVER.
if ($_SERVER['HTTP_ACCEPT_LANGUAGE']) {
}

// Valids:
if (isset($_SERVER[...]) && $_SERVER[...]) { }
if (!empty($_SERVER[...]) && $_SERVER[...]) { }
if (array_key_exists(..., $_SERVER) && $_SERVER[...]) { }
if (isset($_SERVER[...])) { if ($_SERVER[...]) { } }
```

The problem here is to know it we have other variables than could be used like that. For know I just know about `HTTP_ACCEPT_LANGUAGE`, but I think that we have some other keys that should be checked before use to avoid _notices_.

**Comment by kalessil** [comment]
Too many false-positives will be emitted. It's better to use a framework component providing wrappers for request, they are handling those cases normally.
